install/kubernetes/Makefile

```diff
@@ -42,4 +42,13 @@ update-versions:
 clean:
 	$(RM) $(QUICK_INSTALL)
 
-.phony: all clean update-versions
+	$(QUIET)\
+         HUBBLE_PROXY_VERSION=$(HUBBLE_PROXY_VERSION) \
+         HUBBLE_UI_VERSION=$(HUBBLE_UI_VERSION) \
+         MANAGED_ETCD_VERSION=$(MANAGED_ETCD_VERSION) \
+         ETCD_VERSION=$(ETCD_VERSION) \
+         NODEINIT_VERSION=$(NODEINIT_VERSION) \
+         CERTGEN_VERSION=`echo $(CERTGEN_VERSION) | egrep -o '^.*@' | sed 's/@//'` \
+         ../../contrib/release/check-docker-images.sh "v$(VERSION)"
+
+.phony: all check-docker-images clean update-versions
```

Comment:
This is not the same as #13892, the target is missing and half of the versions don't make sense for the v1.7 tree where there was no hubble ui, certgen, hubble proxy, etc.